option optimize_for = LITE_RUNTIME;

message FenjieGallantAnswer
{
 required uint32 result = 1;
 repeated uint32 type = 2;
 repeated uint32 id = 3;
 repeated uint32 num = 4;
}
